harvard-dcu.csl

```diff
@@ -0,0 +1,185 @@
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-use-last="true" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" text-case="lowercase" prefix=" (" suffix=")."/>
+      <substitute>
+        <names variable="editor" prefix="(" suffix=")"/>
+        <text macro="title" suffix="."/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="article-magazine webpage report article-newspaper" match="any">
+        <choose>
+          <if variable="URL">
+            <text value="Available from:" suffix=" "/>
+            <text variable="URL"/>
+            <group prefix=" [" suffix="]">
+              <text term="accessed" text-case="capitalize-first" suffix=" "/>
+              <date variable="accessed">
+                <date-part name="month" suffix=" "/>
+                <date-part name="day" suffix=", "/>
+                <date-part name="year"/>
+              </date>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="italic" suffix="."/>
+      </if>
+      <else>
+        <text variable="title" text-case="capitalize-first"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued" font-variant="small-caps">
+          <date-part name="year"/>
+        </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <group>
+      <label variable="page" form="short" suffix=" "/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```

Comment:
I just checked https://www101.dcu.ie/library/Citing&ReferencingGuide/player.html, and it looks like you need et-al abbreviation for in-text citations for four or more authors, in which case only the first author should be printed. And all authors should be printed in the bibliography.

With that, it's probably clearest to change `<citation et-al-min="3" et-al-use-first="1"` above to `<citation et-al-min="4" et-al-use-first="1"`, and remove `et-al-min="4" et-al-use-first="1"` on line 38. You can delete the `delimiter-precedes-et-al="never"` and `et-al-use-last="true"` attributes on line 28 as well.